editor/editor_export.cpp

```diff
@@ -273,6 +273,8 @@ void EditorExportPlatform::gen_debug_flags(Vector<String> &r_flags, int p_flags)
 }
 
 Error EditorExportPlatform::_save_pack_file(void *p_userdata, const String &p_path, const Vector<uint8_t> &p_data, int p_file, int p_total) {
+	if (p_path.ends_with(".so") || p_path.ends_with(".dylib") || p_path.ends_with(".dll"))
```

Comment:
What was the rationale for ignoring those? Looks like @neikeq needs to un-ignore the `*.dll` for exporting C# assemblies.

Reply:
Can't we ignore these from a custom ExportPlugin? This way I can still add dll files from my `_export_begin` method.

Reply:
I think now this line can be removed altogether - it was necessary for the hacky GDNative export to work (using include filters instead of export plugins).